src/shared/ui/common/region.ts

```diff
@@ -45,8 +45,8 @@ export function createRegionPrompter(
 
     const defaultRegionItem = items.find(item => item.detail === defaultRegion)
 
-    if (defaultRegionItem !== undefined) {
-        defaultRegionItem.description = localize('AWS.generic.recentlyUsed', '(recently used)')
+    if (defaultRegionItem !== undefined && !defaultRegionItem.recentlyUsed) {
+        defaultRegionItem.description = localize('AWS.generic.defaultRegion', '(default region)')
```

Comment:
great to have this fixed! would you mind doing `npm run newChange` and mark it as a "fix", something like:

    Regions quickpick menu shows duplicate "recently used" labels